test/libsyncengine/reconciliation/platform_inconsistency_checker/testplatforminconsistencycheckerworker.cpp

```diff
@@ -158,50 +166,62 @@ void TestPlatformInconsistencyCheckerWorker::testFixReservedNames() {
     node = std::shared_ptr<Node>(new Node(std::nullopt, _syncPal->_remoteUpdateTree->side(), dots, NodeTypeFile,
                                           OperationTypeNone, "0", 0, 0, 12345, nullptr));
     if (PlatformInconsistencyCheckerUtility::instance()->checkReservedNames(node->name())) {
-        // TODO : remove this, files are not renamed anymore
-        node->setValidLocalName(PlatformInconsistencyCheckerUtility::instance()->generateNewValidName(node->name(), PlatformInconsistencyCheckerUtility::SuffixTypeRename));
+        // TODO : to be re-written. Files with reserved names are not renamed anymore, they are blacklisted.
+        //        // TODO : remove this, files are not renamed anymore
+        //        node->setname(PlatformInconsistencyCheckerUtility::instance()->generateNewValidName(
+        //            node->name(), PlatformInconsistencyCheckerUtility::SuffixTypeRename));
     }
-    CPPUNIT_ASSERT(node->name() == dots && node->validLocalName().empty());
+    CPPUNIT_ASSERT(node->name() == dots && node->name().empty());
```

Comment:
For this assert and the similar other ones behind. Can the part after`&&` can be removed as if  `node->name() == dots` , node is not empty?

Reply:
Yes, you ar 100% right. This is a side effect of refactoring I have done on  'validLocalName'.
Its fixed